docs/mkdocs/docs/api/macros/nlohmann_json_namespace_version_namespace.md

```diff
@@ -0,0 +1,46 @@
+# NLOHMANN_JSON_NAMESPACE
+
+```cpp
+#define NLOHMANN_JSON_NAMESPACE_VERSION_NAMESPACE /* value */
+```
+
+This macro evaluates to the name of the version namespace. See [`NLOHMANN_JSON_NAMESPACE`](nlohmann_json_namespace.md).
+If defined to an empty value, the version namespace will be omitted.
+
+## Default definition
```

Comment:
This description sounds like `NLOHMANN_JSON_NAMESPACE`, not `NLOHMANN_JSON_NAMESPACE_VERSION_NAMESPACE`

Reply:
No. That is accurate.

But it might change because determining if a preprocessor macro is defined but empty is near impossible. \*sigh\*